codecov_auth/commands/owner/interactors/save_terms_agreement.py

```diff
@@ -29,6 +30,7 @@ def update_terms_agreement(self, input: TermsAgreementInput):
         self.current_user.terms_agreement = input.terms_agreement
         self.current_user.terms_agreement_at = timezone.now()
         self.current_user.save()
+        self.current_user.customer_intent = input.customer_intent
```

Comment:
`customer_intent` can only be business or personal right? should validate that.

Reply:
Good catch!

Reply:
Will there be any implicit validation provided by `models.TextChoices`?

Reply:
Wait the `customer_intent` is not being saved

Reply:
Also true. I would have to test it myself, but I suspect when you try to do safe with a choice not belonging to text choices, we'd get an error